aggregate/housing_security/rent_stable_three_maintenance.py

```diff
@@ -82,37 +80,22 @@ def load_source_clean_data(indicator: str) -> pd.DataFrame:
     }
     data = pd.read_csv(**read_csv_arg)
     data.columns = [c.replace(".1", "") for c in data.columns]
-    data.columns = [
-        "SBA",
-        "PUMA",
-        "CD Name",
-        "N",
-        "SE",
-        "MOE\n(95% CI)",
-        "CV",
-        "Percent",
-        "Percent SE",
-        "Percent MOE\n(95% CI)",
-    ]
+    data.drop(columns=['SE', 'Percent SE'], inplace=True)
+    data.replace("%", "", regex=True, inplace=True)
+    data.replace(",", "", regex=True, inplace=True)
+    num_cols = ['N', 'MOE\n(95% CI)', 'CV', 'Percent', 'Percent MOE\n(95% CI)']
+    data[num_cols] = data[num_cols].apply(pd.to_numeric, errors="coerce")
 
     denom = pd.read_excel(**read_excel_arg)
     denom["PUMA"] = denom["PUMA"].astype(str)
-    denom.columns = [
-        "SBA",
-        "CD Name",
-        "PUMA",
-        "N",
-        "SE",
-        "MOE\n(95% CI)",
-        "CV",
-    ]
+    if 'Percent' in denom.columns:
```

Comment:
Reading the [documentation on the drop method](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.drop.html), you can pass the `error` parameter as "ignore" and it will only drop the columns it finds, will take this block from four lines to  one